client/scss/components/_button.scss

```diff
@@ -486,12 +507,15 @@ button.button.bicolor .icon-wrapper {
 
 .bicolor {
   border: 0;
-  padding-left: 3.5em;
+  padding-inline-start: 3.5em;
 
   &:before {
     font-size: 1rem;
     position: absolute;
+    // Remove once we drop support for Safari 13.
+    // stylelint-disable-next-line property-disallowed-list
```

Comment:
Just an idea - should we create a mixin? `@include safari { left: 0; }`
Something like that might help to be explicit and also allows us to use a media / supports selector to house the enclosed styles.

Reply:
We’ve discussed this together at the last UI team meeting – I chose to keep this like this because I’d like it to go away in the next + 1 release (August 2022, so we’ll be able to remove this code after the May 2022 release’s code freeze – late April). Hard-coding the styles like this is the pattern that will be the simplest to remove mechanically in 1 month’s time.

Reply:
Agreed.